src/xc_integrator/local_work_driver/device/common/zmat_vxc.hpp

```diff
@@ -16,6 +17,13 @@ void zmat_lda_vxc( size_t        ntasks,
                    XCDeviceTask* tasks_device,
                    device_queue queue );
 
+void zmat_lda_vxc( size_t        ntasks,
```

Comment:
Generic overloads are something we're trying to avoid. Either generalize the original function, or label the functions with the appropriate spin symmetry.